src/compiler/symb.bi

```diff
@@ -2492,6 +2495,10 @@ declare sub symbProcRecalcRealType( byval proc as FBSYMBOL ptr )
 #define	typeSetIsRef( dt ) (dt or FB_DATATYPE_REFERENCE)
 #define	typeUnsetIsRef( dt ) (dt and not FB_DATATYPE_REFERENCE)
 
+#define	typeIsMangleDt( dt ) (((dt and FB_DT_MANGLEMASK) <> 0))
```

Comment:
maybe `typeHasMangleDt` would be a better name, because `typeIsMangleDt` made me think it checks whether the given dtype is one that can be mangled.

Reply:
> maybe `typeHasMangleDt` would be a better name, because `typeIsMangleDt` made me think it checks whether the given dtype is one that can be mangled.

Makes sense to me, OK.